sbin/upload-artifacts

```diff
@@ -113,8 +114,10 @@ s3_upload() {
 	local upload_dir="${S3_URL}/${prod_subdir}${MAYBE_SNAP}"
 	local file
 	if [[ $SNAPSHOT == 1 ]]; then
-		for file in `ls ${prefix}.*${PLATFORM}*.zip ${prefix}.*${PLATFORM}*.tgz 2> /dev/null`; do
-			s3_upload_file $file $upload_dir
+		for platform in ${PLATFORMS[@]}; do
+			for file in `ls ${prefix}.*${platform}*.zip ${prefix}.*${platform}*.tgz 2> /dev/null`; do
+				s3_upload_file $file $upload_dir
+			done
 		done
 	else
 		for file in `ls ${prefix}.*.zip ${prefix}.*.tgz 2> /dev/null`; do
```

Comment:
update here as well

Reply:
not really sure what needs updating here. `prefix` is just `"rejson-oss"`